core/src/main/java/com/adyen/checkout/core/common/CardBrand.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 17/6/2025.
+ */
+package com.adyen.checkout.core.common
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * This class represents a card scheme. The constructor allows for creating a [CardBrand] with a scheme that is not in
+ * the predefined list of [CardType]. Can be used to configure the supported card schemes with
+ * [CardConfiguration.Builder.setSupportedCardTypes].
+ */
+@Parcelize
+data class CardBrand constructor(val txVariant: String) : Parcelable {
+
+    /**
+     * Use this constructor when defining the supported card brand predefined inside [CardType] enum
+     * inside your component
+     */
+    constructor(cardType: CardType) : this(txVariant = cardType.txVariant)
```

Comment:
Wonder if we should still support this constructor. Also thinking if `CardType` should be internal

Reply:
I've made `CardType` restricted since it's used in card module, and removed this constructor. We should also mark it deprecated in v5.